### ISSUE TITLE
Add Linux SDK for building window related extension (DEF-3260)

### ISSUE THREAD

**Issue by britzl** [defold/extender#361]
E.g. the user Tomires wants to build save dialogues for Linux, but the docker container doesn't currently contain such libraries.

The user tried something like this to work "g++ gtktest.cpp `pkg-config --cflags --libs gtk+-3.0`" in order to add the correct cflags and link flags. It seems like a good idea to support.

**Comment by ekharkunov** [comment]
I think task can be closed because custom dialogs can be implemented via extension-umgui. WDYT @britzl

**Comment by britzl** [comment]
Yes, or using https://github.com/andsve/def-diags. Although there is this issue when building that extension for Linux: https://github.com/andsve/def-diags/issues/5
